topcoffea/modules/datacard_tools.py

```diff
@@ -2,6 +2,7 @@
 import gzip
 import topcoffea.modules.HistEFT
 import numpy as np
+import boost_histogram as bh
```

Comment:
Is `bh` actually used?

Reply:
yeah, it's needed to specify the `storage` type of the `hist` histogram when we want to explicitly tell it to save bin weights. So this comes up [here](https://github.com/TopEFT/topcoffea/blob/0d3a827c2bea94c42663d38e8d3ad0508f6dc501/topcoffea/modules/datacard_tools.py#L35).

Reply:
Ah thanks, I didn't scroll the window far enough to see it!